src/core/lib/channel/channel_stack.h

```diff
@@ -159,6 +159,11 @@ typedef struct {
 
   /* The name of this filter */
   const char *name;
+
+  /* For filter with the same \a name, determines which one is kept in the
+   * channel stack. Stringly lesser values take precedence (UNIX priority
```

Comment:
What does "Stringly" mean in this context?  Does it mean that we're printing the values as strings and then doing string comparisons?  If so, why?  Isn't it both easier and more efficient to just do a numeric comparison?